feature-libs/product/configurators/common/src/core/common-configurator-core.module.ts

```diff
@@ -1,4 +1,4 @@
 import { NgModule } from '@angular/core';
 
-@NgModule({})
+@NgModule()
```

Comment:
Do we plan to add anything to this module in the near future?

Reply:
Yes, the store module that is about to be moved to the product lib soon..